randovania/data/json_data/prime1.txt

```diff
@@ -1279,379 +2367,833 @@ Asset id: 1125030300
-  * Normal Door to Quarantine Access B/Dock to Metroid Quarantine B
+> Dock to Quarantine Access B; Heals? False; Spawn Point
+  * Plasma Door to Quarantine Access B/Dock to Metroid Quarantine B
+  > Dock to Save Station Mines C
+      Grapple Beam and Space Jump Boots and Morph Ball and Spider Ball and After Metroid Quarantine B Barrier Lowered
+  > Event (Metroid Quarantine B Barrier Lowered)
+      Scan Visor and Grapple Beam and Space Jump Boots and Morph Ball and Spider Ball
 
 > Dock to Elite Quarters Access; Heals? False
-  * Normal Door to Elite Quarters Access/Dock to Metroid Quarantine B
+  * Plasma Door to Elite Quarters Access/Dock to Metroid Quarantine B
+  > Dock to Save Station Mines C
+      Trivial
 
 > Dock to Save Station Mines C; Heals? False
-  * Normal Door to Save Station Mines C/Dock to Metroid Quarantine B
+  * Plasma Door to Save Station Mines C/Dock to Metroid Quarantine B
+  > Dock to Quarantine Access B
+      Space Jump Boots and After Metroid Quarantine B Barrier Lowered
+  > Dock to Elite Quarters Access
+      Trivial
+  > Pickup (Missile)
+      Shoot Super Missile
+
+> Pickup (Missile); Heals? False
+  * Pickup 85; Major Location? False
+  > Dock to Save Station Mines C
+      Trivial
+
+> Event (Metroid Quarantine B Barrier Lowered); Heals? False
+  * Event Metroid Quarantine B Barrier Lowered
+  > Dock to Quarantine Access B
+      Space Jump Boots
 
 ----------------
 Metroid Quarantine A
 Asset id: 4211416922
-> Dock to Quarantine Access A; Heals? False
-  * Normal Door to Quarantine Access A/Dock to Metroid Quarantine A
+> Dock to Quarantine Access A; Heals? False; Spawn Point
+  * Wave Door to Quarantine Access A/Dock to Metroid Quarantine A
+  > Dock to Elevator Access B
+      All of the following:
+          Morph Ball Bomb and Space Jump Boots and Morph Ball and Spider Ball and After Metroid Quarantine A Barrier Lowered
+          Thermal Visor or X-Ray Visor or Invisible Objects (Intermediate)
+  > Pickup (Missile)
+      All of the following:
+          Power Bomb and Space Jump Boots and Morph Ball and Spider Ball and After Metroid Quarantine A Barrier Lowered
+          Thermal Visor or X-Ray Visor or Invisible Objects (Intermediate)
 
 > Dock to Elevator Access B; Heals? False
-  * Normal Door to Elevator Access B/Dock to Metroid Quarantine A
+  * Ice Door to Elevator Access B/Dock to Metroid Quarantine A
+  > Dock to Quarantine Access A
+      Space Jump Boots and After Metroid Quarantine A Barrier Lowered
+  > Pickup (Missile)
+      All of the following:
+          Power Bomb and Morph Ball and Spider Ball
+          Morph Ball Bomb or Space Jump Boots
+          Thermal Visor or X-Ray Visor or Invisible Objects (Intermediate)
+  > Event (Metroid Quarantine A Barrier Lowered)
+      Scan Visor
+
+> Pickup (Missile); Heals? False
+  * Pickup 86; Major Location? False
+  > Dock to Quarantine Access A
+      Space Jump Boots and After Metroid Quarantine A Barrier Lowered
```

Comment:
Add connection to Front of Spider Track with Space Jump and either [Thermal, Xray or Invisible Objects (Intermediate)]